source/documentation/admin-guide/virt/live-snapshots.html.md

```diff
@@ -26,7 +26,7 @@ An example of a thin provisioned image:
 
 An example of a snapshots:
 
-![](LiveSnapshot_SnapshotImage.png‎ "LiveSnapshot_SnapshotImage.png‎")
+![](LiveSnapshot_SnapshotImage.png "LiveSnapshot_SnapshotImage.png")
```

Comment:
This link still has the redundancy-in-quotes issue... and no prefix for the image either.